.github/workflows/run-tests.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     timeout-minutes: 30
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [16.x, 18.x]
```

Comment:
just one more thing: why not have node 14 in here?

Reply:
Oh, it failed to pass in my CI here: https://github.com/perrin4869/gulp-favicons/actions/runs/4281352674/jobs/7454270512
I didn't look into why really because node 14 becomes EOL next month anyways. If you prefer I can try to see what's breaking node 14 and add it back though

Reply:
I'm fine with it only being done on node16+